src/elements/content-sidebar/additional-tabs/AdditionalTabTooltip.js

```diff
@@ -18,7 +20,18 @@ type Props = {
 };
 
 const AdditionalTabTooltip = ({ children, defaultTooltipText, isFtuxVisible, ftuxTooltipData }: Props) => {
+    const { enabled: isPreviewModernizationEnabled } = useFeatureConfig('previewModernization');
```

Comment:
nit: could we have just passed this prop down as well instead of having to use the useFeatureConfig hook again?

Reply:
@JChan106 take a look when you get back